src/scripts/components/main/globe-navigation/globe-navigation.tsx

```diff
@@ -41,13 +45,48 @@ const GlobeNavigation: FunctionComponent<Props> = ({
     dispatch(setGlobeProjectionAction(newProjection, 2));
   };
 
+  const onLocateMeHandler = () => {
+    setLocationLoading(true);
+
+    if (!navigator.geolocation) {
+      return;
+    }
+
+    navigator.geolocation.getCurrentPosition(
+      position => {
+        const newView = {
+          renderMode: 'globe' as RenderMode.GLOBE,
```

Comment:
cant you use `renderMode: RenderMode.GLOBE` here?

Reply:
nope :(